recipes-support/opencv/opencv_4.6.0.imx.bb

```diff
@@ -276,14 +276,17 @@ SRC_URI += " \
     file://OpenCV_DNN_examples.patch \
 "
 
-PACKAGECONFIG:remove        = "eigen"
-PACKAGECONFIG:append:mx8-nxp-bsp    = " dnn text"
-PACKAGECONFIG_OPENCL        = ""
+PACKAGECONFIG:remove = "eigen"
+
+PACKAGECONFIG:append:mx8-nxp-bsp = " dnn text"
+PACKAGECONFIG:append:mx9-nxp-bsp = " dnn text"
+
+PACKAGECONFIG:append = " ${PACKAGECONFIG_OPENCL}"
```

Comment:
Please align this properly.

Reply:
I actually like it this way because the first line is easier to read, and you can see that it is a simple assignment to `PACKAGECONFIG_OPENCL`, which is aligned itself below the first line.